packages/client/hmi-client/src/components/Header.vue

```diff
@@ -20,9 +22,13 @@ const projectName = 'Name of the project that can be long for clarity and precis
 	<header>
 		<img v-if="isHome" src="@assets/images/logo.png" height="32" width="128" alt="logo" />
 		<img v-else src="@assets/images/icon.png" height="32" width="32" alt="TERArium icon" />
-		<p>
-			<a @onclick="goToHomepage">Projects</a>
-			<span>{{ projectName }}</span>
+		<p v-if="!isHome">
+			<a @click="goToHomepage">Projects</a>
+			<span> {{ projectName }}</span>
+			<!-- Debug -->
```

Comment:
Do we need this debug stuff here still?

Reply:
Yes - we will remove it prior to the final demo build. Right now it is useful for debugging.